### ISSUE TITLE
Hack some nice features I didn't think of

### ISSUE THREAD

**Issue by MrNeRF** [MrNeRF/gaussian-splatting-cuda#10]
Our project seems robust and functional, but there is always room for improvement and innovation. Here are a few potential directions you can pursue:

1. **User Experience Improvement:** Enhance the UI/UX with a more intuitive design, additional customization options, or interactive elements that engage the user in a meaningful way. At the moment this will be basically command line stuff. But I plan that we will provide a viewer
2. **New Functionalities:** Think of a new feature that complements the existing functionalities of the project, offering users additional value that they might not have expected but will likely appreciate.

Simply put:  Hack something delightful and valuable on top of our exiting stuff. Integrate it!

**Comment by andrefpf** [comment]
I think adding a loading bar for the iteration process is a nice improvement for the UX. Probably it is pretty easy too.

**Comment by MrNeRF** [comment]
Should not be hard at all. Of course, depending on how wild you go with the UI.
